ni_measurementlink_generator/ni_measurementlink_generator/template.py

```diff
@@ -104,6 +110,22 @@ def _resolve_service_class(service_class: str, display_name: str) -> str:
     "--directory-out",
     help="Output directory for measurement files. Default: '<current_directory>/<display_name>'",
 )
+@click.option(
+    "-c",
+    "--collection",
+    default="",
+    help="\b\nThe collection that this measurement belongs to. Collection names are specified"\
```

Comment:
Black doesn't use trailing backslashes.

```
PS D:\dev\measurementlink-python\ni_measurementlink_generator> poetry run ni-python-styleguide lint
.\ni_measurementlink_generator\template.py:117:95: BLK100 Black would make changes.
```

Why didn't "Check nimg" catch this?